libs/blocks/tabs/tabs.js

```diff
@@ -68,11 +68,6 @@ function configTabs(config, rootElem) {
     const sel = rootElem.querySelector(id);
     if (sel) sel.click();
   }
-  const tabParam = new URLSearchParams(window.location.search).get('tab');
```

Comment:
What these changes are needed for?

Reply:
It got there by mistake, I removed this change from PR, thank you for noticing this